docs/docs/writing-a-long-lived-client.md

```diff
@@ -44,11 +39,11 @@ if (providedConstraint != null && loadedConstraint != providedConstraint) {
     );
 }
     
-// TODO client interacts with enclave.
+// Code where the client interacts with the enclave.
 
 Files.write(stateFile, enclaveClient.save())
```

Comment:
Here we do not seem to encrypt the byte array, so we are not doing what the warning is suggesting :-)

Reply:
I have shifted the 'warning' call out box to place it after this code. How to encrypt it, it going to be a long piece of code?